official/transformer/transformer_main.py

```diff
@@ -74,32 +77,107 @@ def model_fn(features, labels, mode, params):
     # When in prediction mode, the labels/targets is None. The model output
     # is the prediction
     if mode == tf.estimator.ModeKeys.PREDICT:
+      if params["use_tpu"]:
+        raise NotImplementedError("prediction is not yet supported.")
       return tf.estimator.EstimatorSpec(
           tf.estimator.ModeKeys.PREDICT,
           predictions=output)
 
     logits = output
 
```

Comment:
nit: pre-existing, but why not just have the var name of output be logits above, and avoid this rename?